<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorHairpin</class>
 <widget class="QWidget" name="InspectorHairpin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hairpin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="subtype">
       <item>
        <property name="text">
         <string>crescendo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>decrescendo</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Subtype:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dynamic Range:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Velo. change:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dynRange">
       <item>
        <property name="text">
         <string>Staff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Part</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SYstem</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="veloChange"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="resetDynType">
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons-dark/resetproperty.png</normaloff>:/data/icons-dark/resetproperty.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="resetVeloChange">
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons-dark/resetproperty.png</normaloff>:/data/icons-dark/resetproperty.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="musescore.qrc"/>
 </resources>
 <connections/>
</ui>
